rbac/rook-monitor-mgr-role_binding.yaml

```diff
@@ -1,11 +1,39 @@
-kind: RoleBinding
+---
+kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-monitor-mgr
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: Role
+  kind: ClusterRole
   name: rook-ceph-monitor-mgr
 subjects:
 - kind: ServiceAccount
   name: rook-ceph-mgr
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-monitor
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-monitor
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-system
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-metrics
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-metrics
+subjects:
+  - kind: ServiceAccount
+    # change to the serviceaccount and namespace to use for monitoring
+    name: prometheus-k8s
```

Comment:
This is always in `namespace: openshift-monitoring`.

Reply:
Done